app/src/main/res/values/strings.xml

```diff
@@ -39,6 +40,7 @@
     <string name="main__nodata__favourites">No favourites yet.\nUse the star button of a meme template to add it to the favourites.</string>
     <string name="main__nodata__custom_templates">No custom templates yet.\nPlaced template images here:%1$s</string>
     <string name="main__nodata__saved">No memes saved yet.\nUse the top save button after editing a meme.</string>
+    <string name="main__nodata__hidden">No memes Hidden Yet, Long Press on a meme on hide it</string>
```

Comment:
no need for writing big letters

Reply:
Don't understand, if am correct in the first comment and the hidden bar should not show if the hidden list is empty, then i guess theres no need for this string then, would remove it in that case

Reply:
ehm, yes, not needed  then